src/main/java/uk/gov/ch/repository/update/EntityDataRepository.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.ch.repository.update;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.PagingAndSortingRepository;
+import uk.gov.ch.model.update.EntityData;
+
+public interface EntityDataRepository extends PagingAndSortingRepository<EntityData, String> {
+
+    @Query(value = "select "
+            + "CB.incorporation_number, CBD.email_address "
```

Comment:
no need to select incorporation_number

Reply:
> no need to select incorporation_number

Same reason as my comment above, I can remove no problem.

Reply:
Same reason as my comment above, I can remove no problem.